### DISCUSSION TITLE
Can I combine 'GroupChat' and 'Register_reply'

### DISCUSSION THREAD

**Discussion by Sameerk22** [microsoft/autogen#1781] [General]
I want one of the GroupChat memebers to register reply from the data_stream.
I want  'analyst' to be the one who finds information from data_stream and reply accordingly into the group chat. But currently, I cant find a way to do this.

![code](https://github.com/microsoft/autogen/assets/77271591/62e6d93d-3778-4c20-a95a-316c3eed7ad6)

Referring to these 2 notebooks:
[1] https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_research.ipynb
[2] https://github.com/microsoft/autogen/blob/main/notebook/agentchat_stream.ipynb

**Comment by sonichi** [comment]
In group chat, every participant replies to group chat manager. So the `ceo` as the sender won't trigger the reply function. You can change `ceo` to `firm_manager`.
cc @qingyun-wu @joshkyh @freedeaths
